components/x-audio/src/components/Players.jsx

```diff
@@ -35,7 +37,7 @@ export const ExpandedPlayer = ({
 		<Rewind currentTime={currentTime} updateCurrentTime={updateCurrentTime} />
 		<Forward currentTime={currentTime} updateCurrentTime={updateCurrentTime} duration={duration} />
 		<PlaybackRate rate={playbackRate} onClick={newRate => onPlaybackRateClick({ playbackRate: newRate })} />
-		<div className={classNameMap('audio-player__info__image')}><img alt="dummy"/></div>
+		<Image imageDataSet={imageDataSet} />
```

Comment:
Should this be `imageSrcSet`?

Reply:
Suppose it contains the srcSet so isn't strictly that. Makes sense as is - ignore!